docs/changelog/131261.yaml

```diff
@@ -0,0 +1,10 @@
+pr: 131261
+summary: Enable failure store for log-*-* data streams
+area: Data streams
+type: feature
+issues:
+ - 131105
+highlight:
+  title: Enable failure store for log data streams
+  body: "Closes #131105 Now let's see which tests this breaks 🙂"
```

Comment:
😅

Reply:
Maybe it's proper that it gets to the changelog, as we are still resolving failing tests. 
When I wrote it I meant I expected tests to fail _**locally**_, I didn't think I would need a time machine to resolve failures